pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -204,3 +204,23 @@ func isAntreaPolicyEgressTable(tableID uint8) bool {
 	}
 	return false
 }
+
+// getPacketInTableID returns the OVS table ID in which the packet is sent to antrea-agent. Since L2ForwardOutput is
+// the table where all Antrea-native policies logging packets are sent to antrea-agent, "PacketInTableField" is used
+// to store the real table requiring "sendToController" action. This function firstly parses the direct table where
+// the packet leaves OVS pipeline, then checks whether "PacketInTableField" is set with a valid value or not. The value
+// in the field is returned if yes.
+func getPacketInTableID(pktIn *ofctrl.PacketIn) uint8 {
+	tableID := pktIn.TableId
+	matchers := pktIn.GetMatches()
+	if match := getMatchRegField(matchers, openflow.PacketInTableField); match != nil {
+		tableVal, err := getInfoInReg(match, openflow.PacketInTableField.GetRange().ToNXRange())
+		if err == nil {
+			return uint8(tableVal)
+		} else {
+			// This is not expected, adding a log here for secure.
```

Comment:
what do you mean here by "for secure"?
I would just say, "This is not expected, so we log an error".

Reply:
updated accordingly.

Reply:
updated